### PR TITLE
feat: added links to integrations page in onboarding section

### PR DESCRIPTION
**Pull request by rahulkeswani101**
#### Related Issues / PR's

https://github.com/SigNoz/engineering-pod/issues/1399

#### Screenshots

Before code changes

image

After code changes

image

### CODE REVIEW

**Comment by vikrantgupta25** [comment]
there is box shadow under the `Go To integrations` button which is not present in the design

File: frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss [hunk 1]
```diff
@@ -41,3 +41,19 @@ div[class*='-setup-instructions-container'] {
 .service-name-container {
 	width: 100%;
 }
+
+.intgeration-page-container {
+	background-color: var(--bg-ink-400);
+	border-color: var(--bg-slate-500);
+}
+
```
Comment (vikrantgupta25) [inline frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss:49]: do we need the same styles for the light mode ?

**Comment by vikrantgupta25** [comment]
does the back navigation work as expected after routing to the integrations page ? do we land back exactly here ?

**Comment by rahulkeswani101** [comment]
> does the back navigation work as expected after routing to the integrations page ? do we land back exactly here ?

Yes, it is working as expected.

File: frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss (hunk 1 above)
Comment (rahulkeswani101) [reply frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss:49]: Yes we are giving the same styles and As discussed with @GeekBoySupreme We're not actively prioritizing light mode.

Comment (vikrantgupta25) [reply frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss:49]: cool just make sure that the box is readable enough in light mode

File: frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.tsx [hunk 2]
```diff
@@ -127,6 +129,10 @@ export default function DataSource(): JSX.Element {
 		}
 	};
 
+	const goToIntegrationsPage = (): void => {
+		history.push('/integrations');
```
Comment (vikrantgupta25) [inline frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.tsx:133]: use the `ROUTES` object here else it's one more place to update if we decide to update the route later. `src/constants/routes.ts` can be found in this file

**Review by ahmadshaheer — COMMENTED** [review]
Comparing the [Figma design](https://www.figma.com/design/EgkXVTQCosEQBfNB27YNKo/Signoz-Interface?node-id=3900-739&t=kjc4VST4EEGR8RAD-0) shared by Shuvam and the [after state screenshot](https://private-user-images.githubusercontent.com/175597269/353423266-6fbf71b4-dce0-4dce-a08f-c3fe112486b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NzI1MDMsIm5iZiI6MTcyMjg3MjIwMywicGF0aCI6Ii8xNzU1OTcyNjkvMzUzNDIzMjY2LTZmYmY3MWI0LWRjZTAtNGRjZS1hMDhmLWMzZmUxMTI0ODZiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNVQxNTM2NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjljYTE2Y2M5YzMxYjA2MzM4MDBmNTgxZjIxNGMwMzExY2E2NTVkYzUxM2FiMjQ4MzhmZjAwODYxY2NhZTE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dOo_IUjJCIAxMvRKCGAp_pTAEOO7jkYlio0LwmByfj0) in PR:
- can you change the text color to white, in order to match the design?
- can you remove the border from "Go to integrations" button?
- can you change to text color of "Request a data source" to blue?
- We have a Request integration button in the design screenshot but here we have a textbox and submit button. is this intentional?
- nit: Can you check if the margin top of "Not able to find..." matches the design?

**Comment by rahulkeswani101** [comment]
> * can you change to text color of "Request a data source" to blue?
>* We have a Request integration button in the design screenshot but here we have a textbox and a submit button. is this intentional?
@ahmadshaheer  We have only added a new container on the top containing the Go to Integrations button as a part of this issue. We don't have to change anything else from this page.

**Review by vikrantgupta25 — APPROVED** [review]

**Review by ahmadshaheer — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**